### PR TITLE
Document ability of instructors to remove a response from peer assessment

### PR DESCRIPTION
**Pull request by catong**
This is the PR for TNL-1148. It does not include changes for removing anonymized student IDs.
@symbolist @muzaffaryousaf @explorerleslie can you please review for technical accuracy/intended functionality? Final verification and screenshots to follow when I can run through the workflow on a sandbox.
Sandbox: http://cancellationandusername.m.sandbox.edx.org/

### CODE REVIEW

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 1]
```diff
@@ -109,4 +125,67 @@ For a larger view, click the image so that it opens by itself in the browser win
...
+
+When you remove an inappropriate response from peer assessment, it is
+immediately removed from the pool of submissions available for peer assessment.
+If the inappropriate response has already been sent to other students for peer
+assessment, it is also removed from their queue. However, if any student has
+already graded the inappropriate response, it is counted as one of the
+submissions they have graded.
+
+.. note:: After you remove an inappropriate response from peer assessment, you
+   can choose whether the student who submitted that response is allowed to
```
Comment (explorerleslie) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: This is correct, but when we say "choose", they actually have to go to the Instructor Dashboard > Student Admin tab > enter the student's email/username and problem ID > click Reset Student Attempts. We should explicitly call this out. Let me know if you have questions.

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 2]
```diff
@@ -109,4 +125,67 @@ For a larger view, click the image so that it opens by itself in the browser win
...
+   submit a new response. If you allow the student to submit a replacement
+   response, the student state of the problem is reset. If you do not want to
+   allow the student to submit a replacement response, she receives a grade of
+   zero for each criterion in the assessment and a total grade of zero for the
+   entire submission.
```
Comment (explorerleslie) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: Maybe just say, they receive a zero for the entire submission. Most people won't care about the extra criterion detail and it makes the overall idea harder to parse.

**Comment by explorerleslie** [comment]
Two comments, otherwise :+1: @muzaffaryousaf can you please spin up a sandbox for this so @catong can take final screenshots, or point me to the PR/branch names that I should use to spin up a sandbox myself? Thanks!

**Comment by muzaffaryousaf** [comment]
@explorerleslie do we need to mention the steps to reset the problem state as part of these docs or it's already written.

**Comment by muzaffaryousaf** [comment]
@explorerleslie do we need to mention the cancelled submission scenario in Top X submission leaderboard docs? because technically we would not be showing the submissions with 0 score in Top X submissions.

**Comment by muzaffaryousaf** [comment]
@explorerleslie @catong the sandbox for remove submission and student username to get student info is up at http://muzaffaryousaf.m.sandbox.edx.org/

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: Thanks @explorerleslie I'll add this step, or if it exists elsewhere in the docs I'll reference it.

**Comment by muzaffaryousaf** [comment]
@explorerleslie @catong please let me know when you are done with the testing on this sandbox. I need to deploy content groups PR to the box. Thanks.

**Comment by catong** [comment]
@jhendersonedx FYI

**Comment by catong** [comment]
@muzaffaryousaf -- I have not been able to get student info on this sandbox or get to the point where I can cancel a response. I completed an ORA assignment and self-assessment as honor@example.com, but when I log in again as staff@example.com and enter "honor" as the student username, I get "couldn't find a response for this student".

screen shot 2015-02-03 at 5 21 20 pm

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 3]
```diff
@@ -109,4 +125,67 @@ For a larger view, click the image so that it opens by itself in the browser win
...
+.. note:: Removing a student's submission is an irreversible action. 
+
+When you remove an inappropriate response from peer assessment, it is
+immediately removed from the pool of submissions available for peer assessment.
+If the inappropriate response has already been sent to other students for peer
+assessment, it is also removed from their queue. However, if any student has
+already graded the inappropriate response, it is counted as one of the
+submissions they have graded.
+
+.. note:: After you remove an inappropriate response from peer assessment, you
```
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:163]: @explorerleslie does this change address your comment?

Comment (explorerleslie) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:163]: @catong yes, thanks!

**Comment by catong** [comment]
@mhoeber @lamagnifica @srpearce @muzaffaryousaf @explorerleslie 
Finalized docs based on sandbox experience; added screenshots
(If style not final, will re-take any affected screenshots)

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 4]
```diff
@@ -60,16 +71,21 @@ Accessing information about a specific student has two steps:
 Determine the Student's Course-Specific Anonymized ID
 =====================================================
 
-To determine a student's course-specific anonymized ID, you'll need two .csv spreadsheets from the Instructor Dashboard: the grade report (**<course name>_grade_report_<datetime>.csv**) and the list of course-specific anonymized student IDs (**<course name>-anon-ids.csv**).
+To determine a student's course-specific anonymized ID, you'll need two .csv
+spreadsheets from the Instructor Dashboard: the grade report (**<course
+name>_grade_report_<datetime>.csv**) and the list of course-specific anonymized
```
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:88]: because angle brackets are treated oddly by rst and RTD (they're interpreted as XML), I settled on using braces instead to surround variables in other guides. I'll put the suggestion on the style guide wiki page...

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 5]
```diff
@@ -60,16 +71,21 @@ Accessing information about a specific student has two steps:
...
+student IDs (**<course name>-anon-ids.csv**).
 
 #. In the LMS, click the **Instructor** tab.
 #. On the Instructor Dashboard, click **Data Download**.
-#. On the **Data Download** page, locate the **Data Download** section, and then click **Get Student Anonymized IDs CSV**. A spreadsheet named **<course name>-anon-ids.csv** automatically downloads. Click to open the spreadsheet.
+#. On the **Data Download** page, locate the **Data Download** section, and click **Get Student Anonymized IDs CSV**. A spreadsheet named **<course name>-anon-ids.csv** automatically downloads. Click to open the spreadsheet.
 #. Scroll down to the **Reports** section, and then click **Generate Grade Report**.
```
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:94]: you've lost the expletive-deleted line wrapping a bit again.

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:163]: ...from ^a peer assessment...
oh, I see below that you're not using the indefinite article before peer assessment... either way works!

File: en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst [hunk 6]
```diff
@@ -280,6 +287,10 @@ To calculate the final score, add the median scores for each criterion:
 
 Note, again, that final scores are calculated by criteria, not by individual assessor. Thus the response's score is not the median of the scores that each individual peer assessor gave the response.
```
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst]: can we move away from "thus" and similar archaic terms?
I'd just strike it in this instance.

**Comment by lamagnifica** [comment]
:+1:

File: en_us/students/source/SFD_ORA.rst [hunk 7]
```diff
@@ -264,6 +264,21 @@ If the instructor has included a **Top Responses** section, you can see the high
    :width: 500
 
 
+********************************
+Cancelled Responses
+********************************
+
+If the instructor deems a response that you have submitted to be inappropriate,
+they can cancel that response and remove it from peer grading. In the open
```
Comment (mhoeber) [inline en_us/students/source/SFD_ORA.rst]: instructor/they sing/plural issue

**Comment by mhoeber** [comment]
@catong  looks good.  one comment.  also in the ORA_RemoveSubmission.png, string could use edit. "It" doesn't refer to anything.

**Comment by lamagnifica** [comment]
completely agree with @mhoeber -- is it too late to edit the strings?
(other issue is inconsistent cancel vs. remove as the action)

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 4 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:88]: I'm not to going to worry about edits in this section about obtaining anonymized IDs because it is being removed from the ORA doc (another PR). If it makes sense to do so, in a separate story I'll consolidate that info elsewhere in the B&R guide and will make edits there.

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 5 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:94]: Same comment as above. This section will be gone in another PR.

File: en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst (hunk 6 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst]: Changed it into a note

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:88]: yep, just making a general comment. I added the same info to the wiki.

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst:94]: this was the first place I noticed the line wrapping, but it is a little erratic elsewhere in this PR as well...

**Comment by catong** [comment]
Add updated screenshots for revised UI messaging, finalized xrefs. Done; will merge doc updates when feature releases.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/132
